root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -2932,6 +2963,25 @@ limited_link_type_combinations: [
             expected_clean_url: 'http://www.maniadb.com/album/736792',
        only_valid_entity_types: ['release_group'],
   },
+  // Melon
+  {
+                     input_url: 'https://www.melon.com/album/detail.htm?albumId=11074452',
+             input_entity_type: 'release',
+    expected_relationship_type: ['downloadpurchase', 'streamingpaid'],
+            expected_clean_url: 'https://www.melon.com/album/detail.htm?albumId=11074452',
+  },
+  {
+                     input_url: 'melon.com/album/detail.htm?albumId=11074452',
+             input_entity_type: 'release',
+    expected_relationship_type: ['downloadpurchase', 'streamingpaid'],
+            expected_clean_url: 'https://www.melon.com/album/detail.htm?albumId=11074452',
+  },
+  {
+                     input_url: 'https://m2.melon.com/album/music.htm?albumId=11074452',
+             input_entity_type: 'release',
+    expected_relationship_type: ['downloadpurchase', 'streamingpaid'],
```

Comment:
If I understand correctly, both `downloadpurchase` and `streamingpaid` are auto-selected. If so, a more fine-grain test can be made with the following:
```suggestion
limited_link_type_combinations: [
                                  ['downloadpurchase', 'streamingpaid'],
                                ],
```

Reply:
If I understand it correctly, `expected_relationship_type` tests auto-selection, and `limited_link_type_combinations` checks for all possible variations. I'll add both to my tests :+1: